### PR TITLE
Upload Android binaries

### PR DESCRIPTION
**Pull request by Meakk**

### CODE REVIEW

File: .github/actions/android-ci/action.yml [hunk 1]
```diff
@@ -75,3 +75,16 @@ runs:
...
       run: cmake --build . --parallel 2 --target install
+
+    - name: Upload native library
+      uses: actions/upload-artifact@v4
+      with:
+        path: ./build/lib/libf3d-java.so
+        name: f3d-android-native-${{inputs.arch}}
+
+    - name: Upload Java package
+      if: inputs.arch == 'arm64-v8a' # avoid duplication
```
Comment (mwestphal) [inline .github/actions/android-ci/action.yml:86]: what duplication ?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -359,6 +359,7 @@ jobs:
       uses: ./source/.github/actions/android-ci
       with:
         arch: ${{matrix.arch}}
+        vtk_version: v9.3.0
```
Comment (mwestphal) [inline .github/workflows/ci.yml:362]: why not commit ?

Comment (Meakk) [reply .github/workflows/ci.yml:362]: Vtk::token

File: .github/actions/android-ci/action.yml (hunk 1 above)
Comment (Meakk) [reply .github/actions/android-ci/action.yml:86]: F3d.jar is the same regardless of the architecture. I don't want to pollute artifacts with duplicate archives

Comment (mwestphal) [reply .github/actions/android-ci/action.yml:86]: ok got it

File: .github/workflows/ci.yml (hunk 2 above)
Comment (mwestphal) [reply .github/workflows/ci.yml:362]: ha right

Comment (mwestphal) [reply .github/workflows/ci.yml:362]: please add a link to the issue on top of the step so that we can fix it when we can

**Review by mwestphal — APPROVED** [review]

**Pull request merged**